app/controllers/v3/revisions_controller.rb

```diff
@@ -38,6 +38,7 @@ def fetch_revision(guid, needs_write_permissions: false, needs_secrets_read_perm
     org = space.organization
     resource_not_found!(:revision) unless permission_queryer.can_read_from_space?(space.guid, org.guid)
     unauthorized! if needs_write_permissions && !permission_queryer.can_write_to_space?(space.guid)
+    suspended! if needs_write_permissions && !permission_queryer.is_org_active?(org.guid)
```

Comment:
Add tests to `spec/request/revisions_spec.rb`:
- PATCH /v3/revisions/:revguid

Reply:
suspended.